src/composables/useTour.ts

```diff
@@ -0,0 +1,384 @@
+import { Transaction, useTransactionsStore } from '@/stores/Transactions';
```

Comment:
In this file all the types and all the tour configurations will be placed. 

I am not sure if this is a good use for a composable though, should I place it somewhere else?